Molecular Profiles update (#125)
griffithlab/civic-docs: docs/about/faq.rst

What is a *Variant*?
--------------------

> A genetic variant is a non-exact copy of a biological sequence. This includes single nucleotide variants (SNVs), insertion/deletion events (indels), copy number alterations (CNV's), structural variants (SVs), transcript fusions, and other genomic/molecular events with cancer associations. The vast majority of variants described in CIViC represent mutations at the DNA level, however, alterations that manifest at the RNA, protein, or epigenetic level are also accepted so as long as there is a demonstrated clinical relevance. In general, the concept of a variant in CIViC is very flexible but it is highly preferred that the variant be something that could be measured with some quantitative assay (NGS sequencing, qPCR, etc.). Combinations of one or more variants comprise a Molecular Profile.

Reviewer: ```suggestion
A genetic variant is a non-exact copy of a biological sequence. This includes single nucleotide variants (SNVs), insertion/deletion events (indels), copy number alterations (CNVs), structural variants (SVs), transcript fusions, and other genomic/molecular events with cancer associations. The vast majority of variants described in CIViC represent mutations at the DNA level, however, alterations that manifest at the RNA, protein, or epigenetic level are also accepted so as long as there is a demonstrated clinical relevance. In general, the concept of a variant in CIViC is very flexible, but it is highly preferred that the variant be something that could be measured with some quantitative assay (NGS sequencing, qPCR, etc.). Combinations of one or more variants comprise a Molecular Profile.
```

What is a *Molecular Profile*?
------------------------------

> There is a growing need for clinical annotation of combinations of variants from different genes, such as double hit lymphoma, which is characterized by combinations of mutations in MYC, BCL2, or BCL6, or PIK3CA mutation induced trastuzumab resistance in HER2 overexpressing breast cancer. CIViC addresses this need via use of the Molecular Profile. Molecular Profiles (MPs) are comprised of combinations of one or more variants. A simple Molecular Profile is comprised of one variant, and a complex MP is comprised of a combination of two or more variants. The addition of MPs to CIViC V2 involved an essential change  data model, in that CIViC Evidence Items (EIDs) are now associated to Molecular Profliles instead of directly to variants. This enable clinical annotation of combinations of variants which may not apply to the individual variants in isolation. Annotation of single variants is accomplished by adding Evidence Items (EIDs) simple molecular profiles (e.g. curation of clinical trial showing BRAF V600E sensitivity of vemurafenib in melanoma). Now, annotation of multi variant combinations can be accomplished by adding EIDs to complex Molecular Profiles (e.g. Osimertinib resistance in EGFR L858R and EGFR T790M and BRAF V600E positive long adenocarcinoma). Each MP in CIViC has an associated Molecular Profile page in the user interface containing a curator summary of the MP role in cancer, aliases, links to pages for the individual variants that comprise the MP, and a list of all of the curated Evidence Items which are uniquely associated to the MP. 

Reviewer: ```suggestion
There is a growing need for clinical annotation of combinations of variants from different genes, such as double hit lymphoma, which is characterized by combinations of mutations in MYC, BCL2, or BCL6, or PIK3CA mutation induced trastuzumab resistance in HER2 overexpressing breast cancer. CIViC addresses this need via use of the Molecular Profile. Molecular Profiles (MPs) are comprised of combinations of one or more variants. A simple Molecular Profile is comprised of one variant, and a complex MP is comprised of a combination of two or more variants. The addition of MPs to CIViC V2 involved an essential change to the data model, in that CIViC Evidence Items (EIDs) are now associated to Molecular Profiles instead of directly to variants. This enables clinical annotation of combinations of variants which may not apply to the individual variants in isolation. Annotation of single variants is accomplished by adding Evidence Items (EIDs) to simple molecular profiles (e.g. curation of clinical trial showing BRAF V600E sensitivity of vemurafenib in melanoma). Now, annotation of multi variant combinations can be accomplished by adding EIDs to complex Molecular Profiles (e.g. Osimertinib resistance in EGFR L858R and EGFR T790M and BRAF V600E positive long adenocarcinoma). Each MP in CIViC has an associated Molecular Profile page in the user interface containing a curator summary of the MP role in cancer, aliases, links to pages for the individual variants that comprise the MP, and a list of all of the curated Evidence Items which are uniquely associated to the MP. 
```

Reviewer 2: I can't seem to see the change here. @tmooney

Reviewer 2: Found it. Looks good.

What kinds of Variants belong in CIViC? What does not?
------------------------------------------------------

> Variants, or combinations of variants, are accepted into CIViC provided there is evidence linking them to cancer with some clinical or functional relevance. Relevance to cancer biology alone is NOT sufficient unless there is also some documented relationship between the variants and diagnosis, predisposition, prognosis, functional or oncogenic relevance, or predictive value for a specific treatment. Variants related to diseases other than cancer should not be entered (there is some grey area for cancer-like conditions). CIViC accepts somatic, rare germline, and common germline variants related to cancer (as long as they have clinical or functional relevance), however, most variants in CIViC are somatic mutations. Variants are related to clinical evidence via Molecular Profiles (MPs), which are comprised of combinations of one or more variant, and Evidence Items (EIDs) are curated from literature and abstracts, and associated to MPs. The quality of evidence suggesting clinical relevance of a variant or combination of variants may vary considerably. Before contributing to CIViC please review :doc:`the curation documentation <../curating>` and familiarize yourself with :doc:`the CIViC knowledge model <../model>`

Reviewer: ```suggestion
Variants, or combinations of variants, are accepted into CIViC provided there is evidence linking them to cancer with some clinical or functional relevance. Relevance to cancer biology alone is NOT sufficient unless there is also some documented relationship between the variants and diagnosis, predisposition, prognosis, functional or oncogenic relevance, or predictive value for a specific treatment. Variants related to diseases other than cancer should not be entered (there is some grey area for cancer-like conditions). CIViC accepts somatic, rare germline, and common germline variants related to cancer (as long as they have clinical or functional relevance), however, most variants in CIViC are somatic mutations. Variants are related to clinical evidence via Molecular Profiles (MPs), which are comprised of combinations of one or more variant, and Evidence Items (EIDs) are curated from literature and abstracts, and associated to MPs. The quality of evidence suggesting clinical relevance of a variant or combination of variants may vary considerably. Before contributing to CIViC please review :doc:`the curation documentation <../curating>` and familiarize yourself with :doc:`the CIViC knowledge model <../model>`.
```

What if a drug is shown to have a negative effect on patients with a variant? Or the study was inconclusive?
------------------------------------------------------------------------------------------------------------

> The knowledge model, specifically the Evidence Direction field, is used to indicate whether the study supports or refutes (including inconclusive determinations) any interaction between the Molecular Profle and a clinical action or result. The Clinical Significance field indicates the type of effect the Molecular Profile is determined to have on clinical results, for example, having a positive, negative or neutral/no impact. These descriptions provide human readable interpretations of evidence statements that either support or refute sensitivity or resistance predictions to therapeutics (or other clinical outcomes). For more detailed definitions and specific examples, please review the knowledge model :doc:`here <../model>`.

Reviewer: ```suggestion
The knowledge model, specifically the Evidence Direction field, is used to indicate whether the study supports or refutes (including inconclusive determinations) any interaction between the Molecular Profile and a clinical action or result. The Clinical Significance field indicates the type of effect the Molecular Profile is determined to have on clinical results, for example, having a positive, negative or neutral/no impact. These descriptions provide human readable interpretations of evidence statements that either support or refute sensitivity or resistance predictions to therapeutics (or other clinical outcomes). For more detailed definitions and specific examples, please review the knowledge model :doc:`here <../model>`.
```

My evidence statement disagrees with evidence from another source, should I still add it?
-----------------------------------------------------------------------------------------
Absolutely. CIViC is a forum for discussion of disagreements in the field or literature. Simply log in, go to the Comment tab found on the Evidence, Variant, Molecular Profile, or Gene pages, and discuss this disagreement with the community.

I don't want to add evidence statements, can I still contribute?
----------------------------------------------------------------

> Yes. Evaluation of the literature is a collaborative effort. If you don't want to add new evidence, you can edit or discuss existing evidence. You can also help to make sure the Molceular Proflie Summary is an effective, consise, and accurate summary of the current set of evidence statements and state of knowledge in the field for the Molecular Profile. You can also add promising publications to the source suggestion queue.

Reviewer: ```suggestion
Yes. Evaluation of the literature is a collaborative effort. If you don't want to add new evidence, you can edit or discuss existing evidence. You can also help to make sure the Molecular Profile Summary is an effective, concise, and accurate summary of the current set of evidence statements and state of knowledge in the field for the Molecular Profile. You can also add promising publications to the source suggestion queue.
```
